css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,22 @@
+button {
+  font-size: 2em;
+  padding: 5px 20px;
+}
+
+button:hover {
+  background-color: rgb(0 0 0);
+  color: #0f0;
+}
+
+th {
+  background-color: black;
+  color: white;
+}
+
+tr:nth-child(odd) {
+  background-color: lightgray;
+}
+
+tr:nth-child(even) {
+  background-color: rgb(240 240 240 / 95.2%);
+}
```

Comment:
Lines 20-22 aren't necessary, you can just leave the `background-color` as white (which should be the default)